spec/features/workflow/schedule_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+RSpec.feature "Scheduling an edition" do
+  scenario do
+    given_there_is_an_edition_submitted_for_review
+    when_i_visit_the_summary_page
+    and_i_click_on_schedule_publishing
+    then_i_see_a_default_date_and_time_selected
+
+    when_i_fill_in_a_valid_date
```

Comment:
Could we combine these to make them more descriptive, like 'when_i_fill_in_the_schedule_date_and_time'

Reply:
Changed to `when_i_choose_a_scheduled_date_and_time`